CONTRIBUTING.md

```diff
@@ -420,6 +427,30 @@ $ ./node ./test/parallel/test-stream2-transform.js
 Remember to recompile with `make -j4` in between test runs if you change code in
 the `lib` or `src` directories.
 
+##### Test Coverage
+
+It's good practice to try to improve test coverage, as you add new features
```

Comment:
Nit: no comma after `coverage`

Reply:
Nit: I'd be inclined to frame this more as making sure you're new code is 100% covered by tests. The way it's written now, it might be interpreted as "While you're in there making a change, why don't you add some tests for other uncovered bits of code?" (And I'd dispute that doing so is "good practice".)

Maybe something like this?: "It's a good idea to confirm that any added or changed code is completely covered by tests."